driver-core/src/main/com/mongodb/operation/MixedBulkWriteOperation.java

```diff
@@ -170,12 +168,8 @@ public BulkWriteResult execute(final WriteBinding binding) {
         return withConnection(binding, new CallableWithConnection<BulkWriteResult>() {
             @Override
             public BulkWriteResult call(final Connection connection) {
-                if (bypassDocumentValidationNotSupported(bypassDocumentValidation, writeConcern, connection.getDescription())) {
-                    throw getBypassDocumentValidationException();
-                }
-
-                checkValidWriteRequestCollations(connection, writeRequests);
-
+                checkValidBypassDocumentValidationAndWriteRequestCollations(connection, bypassDocumentValidation, writeRequests,
```

Comment:
These names are going to get even longer if we add a third thing to check for.  How about just

    validateWriteRequests

?